public/javascripts/game.js

```diff
@@ -95,27 +91,59 @@
 			
 			container_rocks.addChild(circle);
 		}
+		
```

Comment:
We might want to consider a food model with a "addAllFoodChildren" method later to clean this up, the file is lengthy as is